### ISSUE TITLE
how to simplify the dataset selection

### ISSUE THREAD

**Issue by yorkie** [alibaba/pipcook#773]
By looking at Amazon ML, just found that we can actually simplify the strategy of selecting the data set. You don’t need to select the training set and the test set, but use here https://docs.aws.amazon.com/machine-learning/latest The Splitting your data method of /dg/splitting-types.html uses some algorithms to dynamically select different datasets.
